frontend/src/component/project/Project/PaginatedProjectFeatureToggles/ProjectFeatureToggles.tsx

```diff
@@ -66,6 +71,19 @@ const Container = styled('div')(({ theme }) => ({
     gap: theme.spacing(2),
 }));
 
+const FilterRow = styled('div')(({ theme }) => ({
+    display: 'flex',
+    flexFlow: 'row wrap',
+    gap: theme.spacing(2),
+    justifyContent: 'space-between',
+}));
+
+const ButtonGroup = styled('div')(({ theme }) => ({
+    display: 'flex',
+    gap: theme.spacing(1),
+    paddingInline: theme.spacing(1.5),
+}));
```

Comment:
This group is there to accomodate the export and column selector buttons which will eventually also move there (according to the sketches)